### DISCUSSION TITLE
the mount point can not mount to docker

### DISCUSSION THREAD

**Discussion by chgyi** [juicedata/juicefs#214] [Q&A] [answered]
i have test the juicefs, but it can not mount into docker .

**Comment by xiaogaozi** [comment]
@chgyi Can you use the [issue template](https://github.com/juicedata/juicefs/issues/new?assignees=&labels=kind%2Fbug&template=bug-report.md) to report more information?

**Comment by davies** [comment]
@chgyi Can you added steps to reproduce the issue?

**Comment by chgyi** [comment]
> @chgyi Can you added steps to reproduce the issue?



> more information?
First i run juicefs format and juicefs mount first , and i works 。 for example i mount if it /home/ubuntu/test_juc/fs
./juicefs mount -d localhost /home/ubuntu/test_juc/fs xxxxxx 


then i run docker run commant , it get the errors .
# docker run -it -v /home/ubuntu/test_juc/fs/:/data -w /data localhost/ubuntu/bin/bash
docker: Error response from daemon: error while creating mount source path '/home/ubuntu/test_juc/fs': mkdir /home/ubuntu/test_juc/fs: file exists.
ERRO[0000] error waiting for container: context canceled

**Comment by chnliyong** [comment, marked as answer]
@chgyi It seems you mount JuiceFS with user `ubuntu`. When mounting JuiceFS with non-root user using default fuse options, the `allow_other` is absent, then `root` has no permission access the mount point.

For this case, docker will create the `/home/ubuntu/test_juc/fs/` directory if it doesn't exist, when it check the existence of the directory (maybe using `stat` system call), `Permission denied` error would return, then docker try to `mkdir` it, `file exists` error returned.

2 Ways to resolve this:
1. Mounting JuiceFS with root user
2. Add `user_allow_other` configuration to `/etc/fuse.conf` where fuse use, then mounting JuiceFS with option `-o allow_other`. Use this with caution as all other accounts can walk to your mount path if the have the parent directory permission.
